src/voip/CallManager.cpp

```diff
@@ -249,7 +261,7 @@ CallManager::hangUp(CallHangUp::Reason reason)
     if (!callid_.empty()) {
         nhlog::ui()->debug(
           "WebRTC: call id: {} - hanging up ({})", callid_, callHangUpReasonString(reason));
-        emit newMessage(roomid_, CallHangUp{callid_, partyid_, "0", reason});
+        emit newMessage(roomid_, CallHangUp{callid_, partyid_, "1", reason});
```

Comment:
So you decided to only support the newer protocol or do you intend to still handle v0 calls when they are incoming?

Reply:
Should I keep the version the call is based on in a variable and do both accordingly?